src/parse-bucket-v2.js

```diff
@@ -60,19 +64,22 @@ const getURL = fileInfo => {
   if (fileInfo.Bucket.includes('.')) {
     return `https://s3.amazonaws.com/${fileInfo.Bucket}/${AWS.util.uriEscapePath(fileInfo.Key)}`
   } else {
-    return `https://${fileInfo.Bucket}.s3.amazonaws.com/${AWS.util.uriEscapePath(fileInfo.Key)}`
+    return `https://${fileInfo.Bucket}.s3.${awsRegion}.amazonaws.com/${AWS.util.uriEscapePath(fileInfo.Key)}`
```

Comment:
this is a breaking change for the stuff i’ve done so far. it **should** be fine, all my data is past this stage and i won’t need to revisit, but if we ever need to revisit/update data we’ve processed before this change, let’s keep in mind that this breaking change landed.